reverse-proxy-nginx.conf

```diff
@@ -37,7 +37,7 @@ http {
       proxy_buffers 4 256k;
       proxy_busy_buffers_size 256k;
       proxy_set_header X-Forwarded-Prefix /api;
-      proxy_pass http://api:3001/;
+      proxy_pass http://http-cache:8080/;
```

Comment:
This would need an update of https://github.com/ecamp/ecamp3/wiki/Reverse-proxy-cache-(API-cache)

Reply:
You mean this pr
- #https://github.com/ecamp/ecamp3/pull/6833

needs this update.
Got no feedback, so i continued